server/core/src/irodsReServer.cpp

```diff
@@ -244,15 +244,7 @@ namespace {
                 logger::delay_server::error((boost::format("rcRuleExecDel failed for %s, stat=%d") %
                     _inp.ruleExecId % status).str());
                 // Establish a new connection as the original may be invalid
-                rodsEnv env{};
-                _getRodsEnv(env);
-                auto tmp_pool = std::make_shared<irods::connection_pool>(
-                    1,
-                    env.rodsHost,
-                    env.rodsPort,
-                    env.rodsUserName,
-                    env.rodsZone,
-                    env.irodsConnectionPoolRefreshTime);
+                auto tmp_pool = irods::make_connection_pool();
```

Comment:
should this be a separate commit/issue?

Reply:
Yup, probably. Removed for later fixing.